### PR TITLE
Task metadata

### PR DESCRIPTION
**Pull request by Procrat**
- [x] Add deadlines
- [x] Add importance
- [x] Add duration
- [x] Add frozen tasks / Waiting-for tasks
- [x] Add possibility to ask Eva what to do

### CODE REVIEW

**Comment by Procrat** [comment]
@Silox Could you review this? ^^

**Comment by Procrat** [comment]
Maybe implement #6 before merging.

File: actions.py [hunk 1]
```diff
@@ -18,6 +18,36 @@ def quit():
...
+            print('Good job! ^_^')
+            task.delete()
+    else:
+        @Action('w', "I'm waiting for something.")
+        def waiting_for():
+            task.waiting_for = ui.ask('What are you waiting for?')
+            print('All right, noted. If you need to poke someone for this or'
+                  ' if you can easily speed this up, do it now.')
+
+        reason = ui.let_choose('Why not?', [waiting_for], none_option='Meh')
```
Comment (Silox) [inline actions.py]: `let_choose` returns a tuple but you only catch 1 return value

File: db.py [hunk 2]
```diff
@@ -38,7 +100,6 @@ class Idea(db.TodoItem):
 
 
 class Reminder(db.Entity):
-    id = PrimaryKey(int, auto=True)
```
Comment (Silox) [inline db.py]: Was this not necessary?

Comment (Procrat) [inline db.py]: Pony automatically generates an integer id as primary key when none other are given

**Comment by Procrat** [comment]
Don't know why this wasn't merged yet.

**Pull request closed without merging**